pgrx/src/callconv.rs

```diff
@@ -265,7 +266,38 @@ argue_from_datum! { 'fcx; Date, Interval, Time, TimeWithTimeZone, Timestamp, Tim
 argue_from_datum! { 'fcx; AnyArray, AnyElement, AnyNumeric }
 argue_from_datum! { 'fcx; Inet, Internal, Json, JsonB, Uuid, PgRelation }
 argue_from_datum! { 'fcx; pg_sys::BOX, pg_sys::ItemPointerData, pg_sys::Oid, pg_sys::Point }
-argue_from_datum! { 'fcx; &'fcx str, &'fcx CStr, &'fcx [u8] }
```

Comment:
Why `CStr` cannot be borrowed?

Reply:
It is removed from this set because it is now covered by the blanket impl of `ArgAbi<'fcx> for &'fcx T where T: ?Sized + BorrowDatum`, because we implement `BorrowDatum` for it here:

https://github.com/pgcentralfoundation/pgrx/blob/7c19edc15802cb462e87362d1cba7625313c454b/pgrx/src/datum/borrow.rs#L88-L104